server/controllers/propertyControllers.js

```diff
@@ -0,0 +1,29 @@
+import express from 'express';
+import bodyParser from 'body-parser';
+import joi from 'joi';
+import PropertyModel from '../models/propertyModels';
+import schema from '../middlewares/validations';
+
+const app = express();
+app.use(bodyParser.json());
```

Comment:
All app instances should be defined from the main entry point (app.js), remove the from here (express,)

Reply:
ok! I noted that